ext/storage_sources/azure_store/azure_storage_source.cpp

```diff
@@ -466,7 +482,8 @@ azure_object_size(
     *sizep = 0;
 
     if ((ret = azure_fs->azure_conn->object_exists(name, exists, size)) != 0) {
-        std::cerr << "azure_object_size: object_exists request to Azure failed." << std::endl;
+        azure_storage->log->log_err_msg(
```

Comment:
A lot of these lines got converted to something like `azure_storage->log->log_X`. Should we instead keep a pointer to the `log`? e.g.

```
auto log = azure_fs->store->log.get();
...
log->log_err_msg(...);
```

Reply:
i've adjusted the majority of the lines to a pointer to the log except where not possible (where the pointer hasnt been created)